### DISCUSSION TITLE
Use Alternate /tmp

### DISCUSSION THREAD

**Discussion by Eireocean** [jhuckaby/Cronicle#504] [Q&A] [answered]
Good Day,

Due to security restrictions we have _noexec_ on _/tmp_. How can we get cronicle to use an alternate path ... ?

Regards

**Comment by jhuckaby** [comment, marked as answer]
Hello,

See previous discussion: https://github.com/jhuckaby/Cronicle/issues/133

> I believe you can set an environment variable to do this.  Cronicle queries the Node.js [os.tmpdir()](https://nodejs.org/api/os.html#os_os_tmpdir) function to get the temp dir path, and I believe that just reads it from the `TMPDIR` global root environment variable (I think -- not sure on this).  So try setting that variable maybe?

> You might be able to specify it in the Cronicle configuration file here: [job_env](https://github.com/jhuckaby/Cronicle#job_env)

Good luck!
